bzt/modules/python.py

```diff
@@ -288,6 +293,12 @@ def build_source_code(self):
 
         test_class.append(test_method)
 
+        if self.appium:
+            imports.text = imports.text.replace("from selenium import webdriver", "from appium import webdriver")
```

Comment:
This is bad strategy. If somebody will change original text, replacement will stop working. Proper approach is to add import conditionally.

Reply:
I agree, I thought to propose a change in PythonGenerator to solve it correctly.
I am going to propose an alternative that solves the problem, I will overwrite the method at the class level and I would handle the conditional logic there.